contracts/v2/ShardingTable.sol

```diff
@@ -157,16 +175,38 @@ contract ShardingTableV2 is Named, Versioned, ContractStatus, Initializable {
                 prevNode.nextIdentityId
             );
 
-        sts.createNodeObject(newNodeHashRingPosition, identityId, prevIdentityId, nextIdentityId, nextNode.index);
-        sts.setIdentityId(nextNode.index, identityId);
+        uint72 index;
+        if (nextIdentityId == NULL) {
```

Comment:
We have written the whole code below in one line: 
```uint72 index = nextIdentityId == NULL ? sts.nodesCount() : nextNode.index;```
But I think that it is better to have it expanded with comments. Just to confirm that it is more optimal to have another if-else insted of calling `sts.nodesCount()`